apps/ember-test-app/tests/integration/components/form/radio-group.gts

```diff
@@ -0,0 +1,53 @@
+import { render, click } from '@ember/test-helpers';
+import { tracked } from '@glimmer/tracking';
+import RadioGroup from '@nrg-ui/ember/components/form/text-area';
+import bind from '@nrg-ui/ember/helpers/bind';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+class Model {
+  @tracked
+  value: string = '';
+}
+
+module('Integration | components | form/radio-group', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders (inline)', async function (assert) {
```

Comment:
Can you add tests to show that `model.value` updates when an item is clicked?
And another to show that changing `model.value` automatically checks the correct item?